bcipy/helpers/system_utils.py

```diff
@@ -49,6 +49,24 @@ def is_battery_powered() -> bool:
     ) and not psutil.sensors_battery().power_plugged
 
 
+def is_screen_refresh_rate_low(refresh: Optional[float] = None) -> bool:
+    """Check if the screen refresh rate is sufficient for BciPy operation.
+
+    Parameters
+    ----------
+        refresh(float) - optional screen refresh rate if already collected. If not provided,
+            the current screen refresh rate is gathered using get_screen_info().
+
+    Returns
+    -------
+    True if the refresh rate is less than 120 Hz; otherwise False.
+    """
+    if refresh is None:
+        refresh = get_screen_info()[-1]
```

Comment:
This might be more readable if `get_screen_info` returned a NamedTuple.

    from typing import NamedTuple
    class ScreenInfo(NamedTuple):
        width: int
        height: int
        rate: float

Reply:
I haven't taken to using NamedTuples often, but I like the suggestion- I'll do it before merging!